lib/presentation/ui/occurrences/occurrences_list_bloc.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:rxdart/rxdart.dart';
+import 'package:vost/common/event.dart';
+import 'package:vost/domain/managers/occurrences_manager.dart';
+import 'package:vost/domain/models/occurrence_model.dart';
+import 'package:vost/presentation/ui/_base/base_bloc.dart';
+
+class OccurrencesListBloc extends BaseBloc {
```

Comment:
See comments on the occurrences_list_page widget for discussion in how we should treat this bloc